src/rust/ensogl/src/display/shape/primitive/glsl/shape.glsl

```diff
@@ -244,6 +251,14 @@ Shape pixel_snap (Shape s) {
     return shape(id,sdf,color);
 }
 
+Shape grow (Shape s, float value) {
+    Id       id    = s.id;
+    BoundSdf sdf   = grow(s.sdf,value);
+    Srgba    color = unpremultiply(s.color);
+    color.raw.a /= s.alpha;
+    return shape(id,sdf,color);
+}
```

Comment:
its 100% ok. just FYI - in most cases the colors will not work ok. Basically we should always apply colors after shapes are done.